src/api/project/model.go

```diff
@@ -0,0 +1,21 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package project
+
+// Project project model
+type Project struct {
```

Comment:
Use the project definition in the common model package directly. The project controller is shared by other components which use the common model

Reply:
This project controller is new created and only used by blob middlewares currently.

If other components want to use this controller, they need to update this model and add fileds they wanted themself.

Reply:
Currently, the artifact handler uses the project manager which should be replaced with project controller. An issue has been opened for this.
We'll move the project model under src/common/models package into a new place during the refactoring for project part.
Is there any special reason you create a new model for project?

Reply:
Do you mean that we use the dao model directly in the controller ?

Reply:
Use the Project of models in controller now.